sbin/fireqos.in.in

```diff
@@ -32,17 +32,27 @@ if [ "$PROGRAM_DIR" = "$0" ]; then PROGRAM_DIR="."; fi
 PROGRAM_PWD="${PWD}"
 declare -a PROGRAM_ORIGINAL_ARGS=("${@}")
 
-# Start defaults before configure
-prefix_POST=/usr
-sysconfdir_POST=/etc
-localstatedir_POST=/var
-libdir_POST=$PROGRAM_DIR
-# End defaults before configure
-for functions_file in $libdir_POST/functions.common.sh
+if [ "@AUTOCONF_RUN@" != "Y" ]
+then
+	echo "Use configure and make, then run 'fireqos'"
+	exit 1
+fi
+
+if [ "$sysconfdir_POST" = "" ]
+then
+	echo "Use make, then run 'fireqos'"
+	exit 1
+fi
+
+for functions_file in functions.common.sh
```

Comment:
No idea. I guess I expected multiple files when I wrote it. I have no issue with eliminating the for,